### ISSUE TITLE
Turbolinks not emitting events for "unsupported" browsers

### ISSUE THREAD

**Issue by balvig** [turbolinks/turbolinks#130]
Perhaps this is "feature not a bug" territory, but _seems_ like before https://github.com/turbolinks/turbolinks/commit/14f07ff12343f77f393ac6c7b571322f3b4e493f, events like `turbolinks:load` and friends would still be fired, which was useful as it meant that all the other JS would still continue to work, even if turbolinks wasn't supported (I'm possible completely wrong here...)

**Comment by sstephenson** [comment]
You’re right, we should fire `turbolinks:load` in unsupported browsers. Not sure how best to handle this right now, but please feel free to explore and open up a pull request if your app needs this functionality.

**Comment by domchristie** [comment]
I feel like the solution in https://github.com/turbolinks/turbolinks/pull/166#issuecomment-305398602 is preferable. It feels a bit odd that a library would emit events when it's not supported.
